nvflare/fuel/hci/tools/admin.py

```diff
@@ -93,10 +100,9 @@ def main():
         client_key=client_key,
         upload_dir=admin_config.get("upload_dir"),
         download_dir=admin_config.get("download_dir"),
-        require_login=admin_config.get("with_login", True),
         credential_type=CredentialType.PASSWORD if admin_config.get("cred_type") == "password" else CredentialType.CERT,
         debug=args.with_debug,
-        overseer_agent=conf.overseer_agent,
+        service_finder=service_finder,
```

Comment:
if no overseer agent, this service finder will be None, 
it will cause logic error in AdminAPI

Reply:
AdminClient already have the service_finder object type check.